modules/rhel-adding-node.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-adding-node-{context}"]
+= Adding a RHEL compute machine to your cluster
+
+You can add compute machines that use Red Hat Enterprise Linux as the operating
+system to a {product-title} {product-version} cluster.
+
+.Prerequisites
+
+* The machine that you run the playbook on must be able to access all of the
+RHEL hosts. You can use any method that your company allows, including a
+bastion with an SSH proxy or a VPN.
+* The `kubeconfig` file for the cluster and the installation program that you
+used to install the cluster are on the machine that you use the run the playbook.
+* You must prepare the RHEL hosts for installation.
+* Configure a user that has SSH access to all of the RHEL hosts.
+* If you use SSH key-based authentication, you must manage the key with an
+SSH agent.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Install the `jq` package.
+
+
+.Procedure
+
+. Extract the pull secret for your cluster.
++
+----
+$ oc -n openshift-config get -o jsonpath='{.data.\.dockerconfigjson}' secret pull-secret | base64 -d | jq .
+----
+
+. Save the pull secret in a file that is named `pull-secret.txt`.
+
+. Create an `openshift-ansible/inventory/hosts` file that defines your compute
+machine hosts and required variables.
++
+----
+[all:vars]
+ansible_user=root <1>
+#ansible_become=True <2>
+
+openshift_kubeconfig_path="~/.kube/config" <3>
+openshift_pull_secret_path="~/pull-secret.txt" <4>
+
+[new_workers] <5>
+mycluster-worker-0.example.com
+mycluster-worker-1.example.com
+----
+<1> Specify the user name to run the playbook as.
+<2> If you do not specify `root` for the `ansible_user`, you must set `ansible_become`
+to `True` and assign the user sudo permissions.
+<3> Specify the path to the `kubeconfig` file for your cluster.
+<4> Specify the path to the file that contains the pull secret for your cluster.
+<5> List each RHEL machine to add to your cluster. Provide a fully-qualified
```

Comment:
This FQDN _must_ be the hostname the cluster will use.  You can't use a public name here and expect the cluster to use a private name to add the host to the cluster.